product_sticker/readme/USAGE.md

```diff
@@ -0,0 +1,10 @@
+Go to Settings \> Technical \> Database Structure \> Product Stickers.
+You can add Stickers to Attributes resulting in different behaviours: -
+If an Image has no Company, it will be available to all Companies - If
+an Image has Available Models, it will be restricted to selected
+Models. - If an Image has no Attribute, it will be available to all
+Attributes - If an Image has no Attribute Value, it will be available to
+all Attribute Values of the Attribute - If an Image has no Category, it
+will be available to all Categories
```

Comment:
issue: This does not render appropriately:

![image](https://github.com/user-attachments/assets/d37ea393-1a27-4e4c-ad6a-7bbaa255c3d2)